content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -16,6 +16,8 @@ what types of Disruptions can happen to Pods.
 It is also for Cluster Administrators who want to perform automated
 cluster actions, like upgrading and autoscaling clusters.
 
+It is most useful if you pod has special requirements to run and not all your nodes provide this environment, eg. some are linux and some are windows. In this case even though the cluster has resources left, the contraints for your pod does not allow it to be scheduled on the other node pool. If all your nodes provide the same environment, you generally don't need Pod Disruption Budgets as Kubernetes will try to keep your given amount of pods available.
```

Comment:
Some suggestions:
- wrap this line
- write Pod in initial caps as it's an object in Kubernetes. Maybe Node too if that feels like it helps the reader.
- fix what I think is a typo: “if you pod has”
- write Linux and Windows as proper nouns
- after “Pod Disruption Budgets”, either add a comma or use a colon and remove “as”